go.mod

```diff
@@ -44,7 +44,7 @@ require (
 	github.com/moby/term v0.0.0-20210619224110-3f7ff695adc6
 	github.com/mrunalp/fileutils v0.5.0
 	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/gomega v1.17.0
+	github.com/onsi/gomega v1.18.0
```

Comment:
Day late dollar short, but please be aware that this is a NOP due to the last line of go.mod:
```
replace github.com/onsi/gomega => github.com/onsi/gomega v1.16.0
```

Reply:
Do you know if we can get rid of that line now?  Any idea why it was added?
@vrothberg Do you know?

Reply:
git history shows
```
    Note that I am pinning github.com/onsi/gomega to v1.16.0 since v1.17.0
    requires go 1.16 which in turn is breaking CI.
```

Reply:
Ok so the same issue we are having with other go vendors. 
Golang 1.16 seems to be in F34 and F35, so I will try a PR to see if this issue is no longer a problem.
@cevich Thoughts?

Reply:
oh, I can check real quick...

Reply:
...A freshly build F35 shows `golang-1.16.13-1.fc35`.  For Ubuntu we download/install 1.17 from upstream.